homework/english/Anastasia_Sheremet/Lecture1/Task_2.md

```diff
@@ -0,0 +1,60 @@
+Домашнее задание по лексике
+Напишите по 1 предложению к каждому слову из лексического блока в 1 уроке (слова есть в презентации).
+
+to boot
+He is booting updates to the server now.
+
+to create
+At the previous lesson we learned how to create a new class.
+
+to connect
+He tryed to connect to the server but every time he received 500 mistake.
+
+to debug
+It took him 20 minutes to debug programm.
+
+to delete
+Unknown hackers deleted important files on his computer.
+
+to develop
+I entered this course to learn how to develop chat bot.
+
+to disable
+You should disable connection after finishing work.
+
+to disconnect
+System administrator disconnected all unverified users from the corporate network. 
+
+to display
+What will be displayed on monitor when I push this button.
+
+to enable
+I recommend you to enable the VPN connection for remote access.
+
+to execute
+What actions can execute your programm?
+
+to implement 
+We decided to implement new technologies in our project.
+
+to install
+I have installed all the necessary software on my computer.
+
+to plug in
+Do you know how to plug in new device?
+
+to restore
+We had to restore the system after a hacker attack.
+
+to uninstall
+Alex decided to uninstall all the software before selling his computer.
+
+to update
+This application is updated almost every month.
```

Comment:
-ed in "updated" is for a Past Simple that means action that been some time ago and finished. 
Most probably its Present Simple "its gets updates"

Reply:
Have you ever had about Passive Voice!!!??? It's OK! I used PRESENT SIMPLE PASSIVE!